### DISCUSSION TITLE
Localization files for Editor.js

### DISCUSSION THREAD

**Discussion by aukc1970** [givanz/Vvveb#236] [General]
Good afternoon! I wanted to ask if your CMS uses the Editor Editor.js and if so, is it possible for this editor to add localization files ?

**Comment by givanz** [comment]
Hi

Editor.js can be installed from https://plugins.vvveb.com/product/editor-js

Translations can be un commented from `/plugins/editor-js/public/admin/init.js` this file is copied/duplicated and loaded from `/public/plugins/editor-js/admin/init.js` after plugin is activated

**Reply by aukc1970** [reply]
Thank you very much!
